airbyte-integrations/bases/base-normalization/unit_tests/test_table_name_registry.py

```diff
@@ -129,26 +129,29 @@ def read_json(input_path: str, apply_function=(lambda x: x)):
 
 
 @pytest.mark.parametrize(
-    "json_path, expected_postgres, expected_bigquery",
+    "json_path, expected_postgres, expected_bigquery, expected_mysql",
```

Comment:
This test is not intended to be exhaustive over the destinations, so it's not mandatory to add your new destination expected field here.

The original intent here was to only unit test simple_name vs nested_hashed_name functions in the table_name_registry